audio/engine.asm

```diff
@@ -537,9 +537,9 @@ _CheckSFX: ; e82e7
 
 PlayDanger: ; e8307
 	ld a, [Danger]
-	bit 7, a
+	bit DANGER_ON_F, a
 	ret z
-	and $7f
+	and $ff - (1 << DANGER_ON_F)
```

Comment:
`and ~(1 << DANGER_ON_F)` or `and ~DANGER_ON`

Reply:
Can't; `~` doesn't do 8-bit negation.

Reply:
weird